profiles/sssd/password-auth

```diff
@@ -12,6 +12,7 @@ auth        sufficient                                   pam_sss.so forward_pass
 auth        required                                     pam_faillock.so authfail                               {include if "with-faillock"}
 auth        optional                                     pam_gnome_keyring.so auto_start                        {include if "with-pam-gnome-keyring"}
 auth        required                                     pam_deny.so
+-auth       sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
if you put this behin pam_deny, then this means pam-deny will already refuse the login attemp.

This should be placed right after the pam_unix.so line above i figure.

Reply:
Done